Jenkinsfile

```diff
@@ -1,118 +1,12 @@
 @Library('socrata-pipeline-library')
 
-String service = 'tileserver'
-String project_wd = '.'
-boolean isPr = env.CHANGE_ID != null
-String lastStage
+Map pipelineParams = [
+  defaultBuildWorker: 'build-worker',
+  deploymentEcosystem: 'marathon-mesos',
+  language: 'scala',
+  languageVersion: '2.12',
+  projectName: 'tileserver',
```

Comment:
Going forward, a sane default for `projectName` might be repository name.

Reply:
Yes, that would be a good default

Reply:
Can we get from `env`?

Reply:
Unfortunately, no